packages/core/src/auth/utils.ts

```diff
@@ -685,7 +685,9 @@ export class ExtensionUse {
 }
 
 /**
- *  Simple readable ID for telemetry reporting
+ * @deprecated
```

Comment:
this should be removed as part of https://github.com/aws/aws-toolkit-vscode/pull/4846 or as a follow up